src/client/studies/TCGA/MedianDistribution.js

```diff
@@ -45,20 +44,32 @@ class MedianDistribution extends React.Component {
       const chartData = this.state.chartData
 
       // Remove any old verticals
-      if (this.verticals) {
-        _.each(this.service.samples, () => chartData.pop())
-      }
+      _.times(this.verticals, () => _.each(this.service.samples, () => chartData.pop()))
```

Comment:
So `verticals` is the count of the number of verticals in the chart right?

Let's just do a `chartData.pop` per vertical instead of bringing in `this.service.samples` into the equation.

I get that it works now because `this.service.samples` is just `['tcga']`, but let's not do that, it's a bit confusing.